source/uwp/AdaptiveCardTestApp/Expected/v1.0.windows-timeline.v1.0.Scenarios.NotificationCard.json

```diff
@@ -0,0 +1 @@
+{"HostConfigHash":"0164943","CardHash":"8dadcfb","Error":null}
```

Comment:
does this card's image supposed to have buttons like other host config versions do?

Reply:
This host config doesn't allow interactivity, so the buttons don't render.

---
In reply to: [178395476](https://github.com/Microsoft/AdaptiveCards/pull/1321#discussion_r178395476) [](ancestors = 178395476)